FlightMatcher/FlightMatcher/City.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  City.swift
+//  FlightMatcher
+//
+//  Created by Daria Korneichuk on 3/31/17.
+//  Copyright © 2017 Dmitry Smolyakov. All rights reserved.
+//
+
+import UIKit
+
+let kLocationFile = URL(string: "Cities.json")
+
+class Location {
+    
+    var ID: Int = 0
+    var city: String = ""
+    var country: String = ""
+    
+    init(with ID: Int, city: String, country: String) {
+        self.ID = ID
+        self.city = city
+        self.country = country
+    }
+
+}
+
+class Request {
+    
+    var ID: Int = 0
+    var from: Location?
+    var to: Location?
+    var date: Date = Date()
+    var flightNumber: Int = 0
+    
+}
+
+public class DataGenerator {
+    
+    public static let shared = DataGenerator()
+    
+    func readJson() -> [Location]? {
+        var locations : [Location]? = [Location]()
```

Comment:
specify variable type like this
```swift
var locations: [Location]? 
```
 not this
```swift
var locations : [Location]? 
```

please pay attention to white spaces